app/views/attach-pvc.html

```diff
@@ -74,7 +74,8 @@ <h2 class="text-center">No persistent volume claims.</h2>
                         Select storage to use.
                       </div>
                       <div ng-if="project && ('persistentvolumeclaims' | canI : 'create')" class="help-block">
-                        Select storage to use or <a ng-href="project/{{project.metadata.name}}/create-pvc" >create storage</a>.
+                        Select storage to use<span ng-if="!outOfClaims"> or <a ng-href="project/{{project.metadata.name}}/create-pvc" >create storage</a>.</span>
+                        <span ng-if="outOfClaims">. Due to exceeded quotas, new storage cannot be create.</span>
```

Comment:
Maybe just "You cannot create new storage since you are at quota."

Although @jwforres is a better wordsmith than me.